interface/src/interfaces/controllers/basic-left-right/BasicLeftRight.vue

```diff
@@ -0,0 +1,12 @@
+<template src="./basic-left-right.html"></template>
+
+<script>
+export default {
+  name: 'BasicLeftRightInterface',
+  emits: ['arrow-left-down', 'arrow-left-up', 'arrow-right-down', 'arrow-right-up']
+}
+</script>
+
+<style lang="scss" scoped>
+  @import './basic-left-right.scss'
```

Comment:
Być może dobrym pomysłem byłoby nie importowanie osobno scss dla każdego interfejsu, a raczej zamknięcie ich pod jednym, głównym scss. Nie musisz tego robić w tym PR, zrobię na to osobne zadanie